docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+services:
+  rabbitmq-broker:
+    image: rabbitmq:3-management
+    restart: always
+    environment:
+      RABBITMQ_DEFAULT_USER: guest
```

Comment:
Is it possible to get these credentials from Env?

Reply:
Well yes, but I see no need for it since we use the docker compose only for testing and not for production.

Reply:
Since it's only for testing purposes I think we could resolve the following conversation.